ibu-imager/clusterinfo/clusterinfo.go

```diff
@@ -54,18 +57,40 @@ func (m *InfoClient) CreateClusterInfo(ctx context.Context) (*ClusterInfo, error
 	if err := m.client.Get(ctx, types.NamespacedName{Name: "version"}, clusterVersion); err != nil {
 		return nil, err
 	}
+
 	installConfig, err := m.getInstallConfig(ctx)
 	if err != nil {
 		return nil, err
 	}
+
+	ip, err := m.getNodeInternalIP(ctx)
+	if err != nil {
+		return nil, err
```

Comment:
Errors should be wrapped

Reply:
Hmm, you think we should wrap each error? with some better string? 
The question is where better to wrap it inside caller or callee

Reply:
I think it should be wrapped everywhere to give context, otherwise you get a vague error and you have no idea where it came from